README.md

```diff
@@ -79,6 +79,14 @@ Run `make archive` from the root directory to generate `.xcframeworks` for each
 
 Additional documentation for usage and SDK architecture can be found under the [Documentation](Documentation/README.md) directory.
 
+## Related Projects
+
+[AEP SDK Sample App for iOS](https://github.com/adobe/aepsdk-sample-app-ios) contains iOS sample apps for the AEP SDK. Examples are provided for both Objective-c and Swift implementation.
+
+[AEP SDK Sample Extension for iOS](https://github.com/adobe/aepsdk-sample-extension-ios) contains a sample implementation of an iOS extension for the AEP SDK. Example implementations are provided for both Objective-c and Swift.
+
+[AEP SDK Compatibility for iOS](https://github.com/adobe/aepsdk-compatibility-ios) contains code required to migrate from the existing ACP SDK to this AEP SDK without making implementation changes. While updating your implementation is recommended, this option allows you to upgrade to the AEP SDK with minimal effort.
```

Comment:
@shalehaha pointed out that we should be more specific that this only bridges ACPCore to AEPCore and "3rd party extension" APIs, it doesn't handle first party extensions (e.g. ACPAnalytics). maybe this would be better?

```
contains code that bridges `ACPCore` and 3rd party extension implementations into the AEP SDK runtime.
```